server/mailet/mailets/src/test/resources/rspamd-config/actions.conf

```diff
@@ -0,0 +1,29 @@
+# Actions settings
```

Comment:
Why is this file needed?

Reply:
The default rejects score of rspamd is 15. 
It will take more time if try to find the mail sample that has a score > 15. 
We already have a mail sample that has scored is 14.25, so it is easier to custom rspamd to decrease the threshold of action


More: 
https://github.com/apache/james-project/pull/1088#issuecomment-1195294537
https://github.com/apache/james-project/pull/1093#issuecomment-1198874573